model_security_adjust_oaw/reports/sale_order_reports.xml

```diff
@@ -135,7 +135,7 @@
                         </div>
                     </div>
 
-                    <p t-field="o.note"/>
+                    <p t-field="o.quot_report_note"/>
```

Comment:
actually I am little bit concerned about this field since it is defined in one of my custom modules.  a custom module that is not in the dependency list of that module. But I could make it work even without mentioning that module in this ones __open__erp